host/class/uvc/usb_host_uvc_2/examples/basic_uvc_stream/main/basic_uvc_stream.c

```diff
@@ -0,0 +1,312 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stdio.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include "esp_system.h"
+#include "esp_log.h"
+#include "esp_err.h"
+
+#include "esp_vfs_fat.h"
+#include "sdmmc_cmd.h"
+#include "driver/sdmmc_host.h"
+#include "sd_pwr_ctrl_by_on_chip_ldo.h"
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+#include "freertos/queue.h"
+
+#include "usb/usb_host.h"
+#include "usb/uvc_host.h"
+#include "esp_private/uvc_stream.h"
+
+#define EXAMPLE_USB_HOST_PRIORITY   (15)
```

Comment:
What will be, if the priority of frame handling task will be lower that `usb_host_lib` priority?  
Is this a requirement for frame handling tasks? (something like: their prio should higher than `usb_host_lib` to handle the frame buffer sooner that usb will provide the next one).

Reply:
This indeed might be a problem, I'll double check

Reply:
I made couple of tests and the priorities should not matter.

There was only one bug, if you tried to install the UVC driver from high priority task. 

Fixed by https://github.com/espressif/esp-usb/pull/84/commits/104b5c141257423180efecb671197e12ac8518e8